internal/pkg/agent/application/monitoring/process.go

```diff
@@ -33,6 +53,18 @@ func processHandler(coord *coordinator.Coordinator, statsHandler func(http.Respo
 			return statsHandler(w, r)
 		}
 
+		metricsPath := vars["metricsPath"]
+		if _, ok := redirectPathAllowlist[metricsPath]; ok {
+			if isProcessRedirectable(id) {
+				return redirectToPath(w, r, id, metricsPath, operatingSystem)
+			}
+			return errorfWithStatus(http.StatusNotFound, "endpoint not found")
```

Comment:
[Suggestion | question]
Is it really not found? What I'm thinking is: there is a redirect for this path, but the process isn't allowed to redirect. Thus, perhaps the error message could explain better the problem?